sentinelhub/aws_safe.py

```diff
@@ -237,7 +237,7 @@ def get_safe_struct(self):
         if self.data_collection is DataCollection.SENTINEL2_L2A:
             for mask in AwsConstants.CLASS_MASKS:
                 for resolution in [AwsConstants.R20m, AwsConstants.R60m]:
-                    if self.baseline <= '02.06':
+                    if self.baseline <= '02.06' or self.baseline == '00.01':
```

Comment:
It seems that everywhere else baseline `00.01` is handled the same as baselines `02.07` and above. Only here baseline `00.01` is handled the same as baselines `02.06` and below. Is there a reason for this?

Note that this results into a different naming of 4 metadata masks.

- this is what we get at the moment:
```
T33UUQ_20170414T101021_SNW_20m.jp2
T33UUQ_20170414T101021_SNW_60m.jp2
T33UUQ_20170414T101021_CLD_20m.jp2
T33UUQ_20170414T101021_CLD_60m.jp2
```
- this is what we could get for baselines `>=02.07`
```
MSK_SNWPRB_20m.jp2
MSK_SNWPRB_60m.jp2
MSK_CLDPRB_20m.jp2
MSK_CLDPRB_60m.jp2
```
- and this is what we could get for baselines `<=02.06` (if everything would be consistent with those):
```
L2A_T33UUQ_20170414T101021_SNW_20m.jp2
L2A_T33UUQ_20170414T101021_SNW_60m.jp2
L2A_T33UUQ_20170414T101021_CLD_20m.jp2
L2A_T33UUQ_20170414T101021_CLD_60m.jp2
```

So the current naming of these masks for `00.01` is not consistent neither with the old baselines nor with the newer.

Reply:
You are completely right.

I just changed this part to treat the reprocessed files as baselines `02.07` and above.

Reply:
Excellent, all clear now.